src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/expense/ExpenseController.java

```diff
@@ -0,0 +1,63 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.expense;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.expense.ExpenseService;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseResponseDTO;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("expense")
+public class ExpenseController {
+
+    private final ExpenseService expenseService;
+
+    public ExpenseController(ExpenseService expenseService) {
+        this.expenseService = expenseService;
+    }
+
+    @PostMapping
+    public ResponseEntity<?> create(@Valid @RequestBody ExpenseCreateDTO expenseCreateDTO) {
+        try {
+            ExpenseResponseDTO savedExpense = this.expenseService.saveExpense(expenseCreateDTO);
+            return ResponseEntity.ok(savedExpense);
+        } catch (IllegalArgumentException e) {
+            return ResponseEntity.badRequest().body(e.getMessage());
+        } catch (Exception e) {
+            return ResponseEntity.internalServerError().body(e.getMessage());
+        }
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> find(@PathVariable UUID id) {
```

Comment:
Altera esse `find`para `findById`para manter o padrão com o IncomeController
```suggestion
    public ResponseEntity<?> findById(@PathVariable UUID id) {
```